setup_sbt.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+: "
+This script setups your environment for Scala SBT connected to JFrog. It creates files:
+- $HOME/.sbt/.credentials
+- $HOME/.sbt/repositories
+- $HOME/.sbt/1.0/plugins/credentials.sbt
+"
```

Comment:
What does this syntax do?

Reply:
It's just a comment. It describes what this script do. It can be removed if you think it's uselees.

Reply:
Nope, comments are fine